src/lib/tracker.js

```diff
@@ -0,0 +1,26 @@
+import trackingTool from '@sap_oss/automated-usage-tracking-tool'
```

Comment:
This file makes the file src/tracker/tracker.js obsolete, remove the latest one.